recipes/mpfr/all/test_v1_package/test_package.c

```diff
@@ -0,0 +1,32 @@
+/* example was taken from: https://www.mpfr.org/sample.html */
```

Comment:
This file can be deleted, and re-use the same from test_package/ folder by test_v1_package/cmakelists.txt

Reply:
Deleted.